private/projects/pdk-monorepo-project.ts

```diff
@@ -95,14 +95,6 @@ export class PDKMonorepoProject extends NxMonorepoProject {
           "@aws-prototyping-sdk/*/license-checker/*",
           "@aws-prototyping-sdk/*/oss-attribution-generator",
           "@aws-prototyping-sdk/*/oss-attribution-generator/*",
-          "@aws-prototyping-sdk/open-api-gateway/openapi-types",
-          "@aws-prototyping-sdk/open-api-gateway/openapi-types/*",
-          "@aws-prototyping-sdk/open-api-gateway/fs-extra",
-          "@aws-prototyping-sdk/open-api-gateway/fs-extra/*",
-          "@aws-prototyping-sdk/open-api-gateway/lodash",
-          "@aws-prototyping-sdk/open-api-gateway/lodash/*",
-          "@aws-prototyping-sdk/open-api-gateway/log4js",
-          "@aws-prototyping-sdk/open-api-gateway/log4js/*",
```

Comment:
Looks like the bundled deps for `open-api-gateway` aren't being picked up?

Reply:
Oh weird, they were added, must have changed something with cleanup refactor. Ok will fix this, good catch!

Reply:
@cogwirrel Actually this is correct. They are removed from `pdk-monorepo-project`, but still in `package.json`. Because of the "automatic nohoist for bundles" they are still in package.json, and no longer need to be maintained in the projen project :) 

https://github.com/aws/aws-prototyping-sdk/pull/244/files/f26185dc614c6194b9a6933ea2ded257a8a99467#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

![image](https://user-images.githubusercontent.com/464119/207181864-5e66964e-2743-44a0-978b-d7b377e9a5d6.png)

> The cloudscape stuff is also not removed as per the screenshot, just moved higher in the list - see actual package.json file

Reply:
ahh yes of course! thank you! :D